Tests/SonarScanner.Shim.Tests/PropertiesWriterTest.cs

```diff
@@ -97,9 +98,9 @@ public void PropertiesWriterToString()
             using (new AssertIgnoreScope()) // expecting the property writer to complain about the missing file
             {
                 var writer = new PropertiesWriter(config);
-                writer.WriteSettingsForProject(productCS, productCoverageFilePath);
-                writer.WriteSettingsForProject(productVB, null);
-                writer.WriteSettingsForProject(test, null);
+                writer.WriteSettingsForProject(productCS, productCoverageFilePath, productTrxPath);
```

Comment:
What happens if the user has also supplied a value for _sonar.cs.vstest.reportsPaths_ on the command line?